.travis.yml

```diff
@@ -7,4 +7,9 @@ cache:
 rvm:
   - 2.2.4
   - 2.3.0
-  - ruby-head
\ No newline at end of file
+  - ruby-head
+
+matrix:
+  allow_failures:
+    - rvm: ruby-head
+
```

Comment:
@sarangan12 Since allowing failures is not flagging them, we're likely to miss them, so I'd suggest removing ruby-head from here. Setting up Jenkins as a warning mechanism should be enough for us.